.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: nixbuild/nix-quick-install-action@v17
-      - run: nix-instantiate release.nix --arg supportedSystems '[ builtins.currentSystem ]' --restrict-eval -I . --allowed-uris 'https://github.com/NixOS/nixpkgs https://github.com/input-output-hk https://github.com/NixOS/nixpkgs-channels' --option trusted-public-keys "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" --option substituters "https://hydra.iohk.io https://iohk.cachix.org https://cache.nixos.org/" --show-trace
+      - run: nix-instantiate obsolete.release.nix --arg supportedSystems '[ builtins.currentSystem ]' --restrict-eval -I . --allowed-uris 'https://github.com/NixOS/nixpkgs https://github.com/input-output-hk https://github.com/NixOS/nixpkgs-channels' --option trusted-public-keys "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" --option substituters "https://hydra.iohk.io https://iohk.cachix.org https://cache.nixos.org/" --show-trace
```

Comment:
All changes to release.nix, ci.nix and lib/ci.nix are needed to test the new flake setup on hydra. 
The release.nix has been renamed to obsolete.release.nix, so that the "old" release.nix CI won't be triggered, just the new flake-based one.

Reply:
I was assuming we'd do the CI switchover when we also switch over to the std-based setup?

Reply:
Does this mean that our CI is going to start using the std-based setup once we merge this? That seems risky if it isn't finished!

Reply:
My comment wasn't very clear, sorry.

We've added a new jobset called plutus-flaked.
This means that each PR to plutus repo triggers two CI runs, one for release.nix and one for flake.nix
In order to speed things up I've renamed release.nix to obsolete.release.nix, but this change won't be merged. 
Once this PR is ready, I'll rename obsolete.release.nix -> release.nix

Reply:
(I've renamed it so that this PR won't trigger the old (release.nix) CI jobset)